<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainUI</class>
 <widget class="QWidget" name="mainUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="listWidget_frlist">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>131</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_message">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>261</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_seed">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>340</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_message">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>261</width>
     <height>70</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
